src/applications/vaos/components/PastAppointmentsList.jsx

```diff
@@ -44,13 +45,16 @@ export class PastAppointmentsList extends React.Component {
       prevProps.appointments.pastStatus === FETCH_STATUS.loading &&
       this.props.appointments.pastStatus === FETCH_STATUS.succeeded
     ) {
+      focusElement('#pastAppts');
```

Comment:
I don't think we want to do this, this will move focus off the tab when the data loads.

Reply:
Ok. So on initial load, stay on the tab.

Reply:
Yeah, after the initial data load, whenever we fetch new data, we want to set the focus to "showing results" text.